packages/manager/apps/nasha-react/src/pages/details/[serviceName]/engagementUpToContent.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import {
+  OsdsChip,
+  OsdsDivider,
+  OsdsText,
+} from '@ovhcloud/ods-stencil/components/react/';
+import { OdsThemeColorIntent } from '@ovhcloud/ods-theming';
+
+interface EngagedUpToContentProps {
+  dateEngagementUpTo: string | React.ReactNode;
```

Comment:
```suggestion
  dateEngagementUpTo: React.ReactNode;
```

string is a ReactNode